File: cs50x/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
   int n;
   do
   {
        n = get_int("Height: ");
   }
   while (n < 1 || n > 8);

   // For each row
    for (int i = 0 ; i < n ; i++)
    {
        // For each column
        for (int j = n -1; j > i; j--)
        {
            // For print space
            printf(" ");
        }
        for (int x = -1; x < i; x++)
    {
            // Print a brick
            printf("#");
        }

        // Move to next row
        printf("\n");
    }

}

File: cs50x/sentimental-mario-less/mario.py
# TODO

from cs50 import get_int

while True:
    height = get_int("height: ")
    if height > 0 and height < 9:
        break

for i in range(0, height, 1):
    for j in range(0, height, 1):
        if (i+j < height-1):
            print(" ", end="")
        else:
            print("#", end="")
    print()